google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkCommitBatch.java

```diff
@@ -117,6 +114,13 @@ ApiFuture<Void> bulkCommit() {
         executor);
   }
 
+  private BatchWriteRequest buildBatchWriteRequest() {
```

Comment:
nit: I think this method does not need to exist, and can just be inlined at the single call site (where it was previously inlined).

Reply:
There is some value in separating out request builder logic.

The `builder` variable is encapsulated in the scope of method, and any interaction with built request is through an immutable object. This lowers the mental burden of reasoning about `commit()`.

Also breaking larger code blocks down into smaller more easily digestible blocks helps readability. go/java-practices/methods#length

Reply:
Ok sounds good.